src/views/Event.vue

```diff
@@ -47,15 +50,15 @@
               </tr>
               <tr>
                 <td>created on</td>
-                <td>{{event.created_on}}</td>
+                <td>{{event.created_on | luxon }}</td>
               </tr>
               <tr>
                 <td>starts on</td>
-                <td>{{event.starts_on}}</td>
+                <td>{{event.starts_on | luxon }}</td>
               </tr>
               <tr>
                 <td>ends on</td>
-                <td>{{event.ends_on}}</td>
+                <td>{{event.ends_on | luxon }}</td>
```

Comment:
what if the ends_on is not set or it is zeroed?

Reply:
shall I just remove it for now or you mean format it differently?

Reply:
the back-end will always report a date, but if it wasn't specifically set it will be something like 0000-00-00 , maybe you can check if the ends_on is before the start_on and in that case display ony a `-`